kerckhoff/packages/serializers.py

```diff
@@ -172,3 +174,82 @@ def get_version_data(self, obj: Package):
     class Meta(PackageSerializer.Meta):
         fields = PackageSerializer.Meta.fields + ("version_data",)
         read_only_fields = PackageSerializer.Meta.read_only_fields + ("version_data",)
+
+class PackageInfoSerializer(serializers.ModelSerializer):
+    package_set = serializers.StringRelatedField()
+    latest_version = serializers.StringRelatedField()
+    description = serializers.SerializerMethodField()
+    folder_id = serializers.SerializerMethodField()
+    folder_url = serializers.SerializerMethodField()
+    metadata = serializers.SerializerMethodField()
```

Comment:
I think just put all the obj.metadata fields into metadata instead of leaving it as an empty dictionary. And remove the ones from the top level e.g. google drive folder id and folder url

Reply:
Done.